src/components/LinkManager/FavoriteButtonList/FavoriteButtonList.tsx

```diff
@@ -0,0 +1,27 @@
+import { Favorite } from "@/src/constants/types";
+import FavoriteButton from "./FavoriteButton/FavoriteButton";
+import styles from "./FavoriteButtonList.module.scss";
+
+interface FavoriteButtonList {
+  favorites?: Favorite[];
+  handlefavoriteClick: (favorite?: Favorite | null) => void;
```

Comment:
```suggestion
  handleFavoriteClick: (favorite?: Favorite | null) => void;
```

Reply:
컴포넌트가 props로 이벤트 핸들러를 받을떄는 on- 으로 표현해주는게 일반적입니다~
```suggestion
  onFavoriteClick: (favorite?: Favorite | null) => void;
```

Reply:
앗 또 실수해버렸네요 감사합니다!